client/question/services/FeedbackGeneratorServiceSpec.js

```diff
@@ -171,61 +167,98 @@ describe('FeedbackGeneratorService', function() {
   });
 
   describe('_getCorrectnessTestFeedback', function() {
-    it([
-      'should return feedback on a user function that ',
-      'doesn\'t pass all correctness tests'
-    ].join(''), function() {
-      var testCase = TestCaseObjectFactory.create({
-        input: 'cat',
-        allowedOutputs: ['a']
-      });
-
-      var paragraphs = FeedbackGeneratorService
-        ._getCorrectnessTestFeedback(null, testCase, 'output').getParagraphs();
-      expect(paragraphs.length).toEqual(5);
-      expect(paragraphs[0].isTextParagraph()).toBe(true);
-      expect(paragraphs[0].getContent()).toBe(
-        "Your code produced the following result:");
-      expect(paragraphs[1].isCodeParagraph()).toBe(true);
-      expect(paragraphs[1].getContent()).toBe(
-        "Input: \"cat\"\nOutput: \"output\"");
-      expect(paragraphs[2].isTextParagraph()).toBe(true);
-      expect(paragraphs[2].getContent()).toBe(
-        "However, the expected output is:");
-      expect(paragraphs[3].isCodeParagraph()).toBe(true);
-      expect(paragraphs[3].getContent()).toBe("\"a\"");
-      expect(paragraphs[4].isTextParagraph()).toBe(true);
-      expect(paragraphs[4].getContent()).toBe(
-        "Could you fix this?");
+    var sampleInputTestSuite = {
+      id: 'SAMPLE_INPUT',
+      testCase: {
+        input: 'Hello, John',
+        allowedOutputs: ['olleH, nhoJ']
+      }};
```

Comment:
Optional: consider dropping the last `};` to the next line so that it matches the indentation level of its corresponding start brace. Ditto below.

Reply:
Changed.